answers/README.md

```diff
@@ -0,0 +1,165 @@
+# ANSWERS
+
+| Method | Route                        | Description                                                    |
+| ------ | ---------------------------- | -------------------------------------------------------------- |
+| POST   | [/answers](#post---answers)  | it will be to submit answer by the peers                       |
+| PATCH  | [/answers](#patch---answers) | for updating the answers(will primarily be using for approval) |
+| GET    | [/answers](#get---answers)   | it will be to get the answers                                  |
+
+### POST - /answers
+
+It will be used for peers to answer questions.
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - **`answer=[STRING]`** required - (The answer text)
+  - **`answeredBy=[STRING]`** required (The ID of the peer who answered the question.)
+  - **`eventId=[STRING]`** required (The ID of the event where the question was asked.)
+  - **`questionId=[STRING]`** required (The ID of the question for which the answer is.)
+- **Headers**
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - `none`
+- **Middlewares**
+  - `none`
+- **Success Response:**
+  - **Status Code:** 201 CREATED
+  - **Content:**
+  ```json
+  {
+    "message": "Answer submitted successfully",
+    "data": {
+      "id": "<STRING>",
+      "answer": "<STRING>",
+      "is_approved": "<BOOLEAN>", //so that HOST | MODERATOR can filter out the answers which has to be shown in word cloud
+      "approved_by": "<RDS_USER_ID>", //can be of HOST | MODERATOR
+      "question_id": "<STRING>", //FOREIGN KEY POINTING TO QUESTION COLLECTION
+      "answered_by": "<STRING>", //ID OF THE RESPONDER OF THE QUESTION(FROM PEERS COLLECTION)
+      "event_id": "<STRING>", //ID OF THE EVENT IN WHICH ANSWER IS GIVEN
+      "timestamps": {
+        "created_at": "<FIREBASE_TIMESTAMP>",
+        "updated_at": "<FIREBASE_TIMESTAMP>"
+      }
+    }
+  }
+  ```
+- **Error Response:**
+  - **Code:** 500 INTERNAL_SERVER_ERROR
+    - **Content:**
+    ```json
+    {
+      "statusCode": 500,
+      "error": "Internal Server Error",
+      "message": "An internal server error occurred"
+    }
+    ```
+
+### PATCH - /answers
+
+It will be used to update the answers.
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - **`isApproved=[BOOLEAN]`** optional
```

Comment:
- can we reject an answer?

Reply:
yes updated the api contracts 👍, thanks for pointing out